library/RAVE/src/main_audio_face_detection.py

```diff
@@ -1,47 +1,102 @@
+# import json
```

Comment:
Is this dead code? We souldn't merge to master big chunks of comments like these.